scss/cayman.scss

```diff
@@ -59,19 +71,19 @@ a {
 .btn {
   display: inline-block;
   margin-bottom: 1rem;
-  color: rgba($button-text-color, 0.7);
-  background-color: rgba($light-color, 0.08);
-  border-color: rgba($light-color, 0.2);
+  color: rgba($btn-text-color, 0.7);
+  background-color: rgba($btn-bg-color, 0.08);
+  border-color: rgba($btn-border-color, 0.2);
```

Comment:
We're still using a bunch of `rgba()` calls on hex colors. Let's define these with the full `rgba` definition.